docs/build_and_setup.md

```diff
@@ -98,7 +98,7 @@ make -j`nproc`
 make install
 ```
 
-This generates the guestlib and API server codes in `ava/cava/demo_nw` and
+This generates the guestlib and API server codes in `ava/cava/demo` and
```

Comment:
The generated code is still inside `demo_nw`.
It's planned to put all generated code into `ava/cava/codegen` and remove the `_nw` suffix.